sqlwhat_ext.py

```diff
@@ -31,3 +32,33 @@ def check_result2(state, col_names = None, sort = False, match = 'exact'):
 
     # return state in case another SCT is used after
     return state
+
+@state_dec
+def check_result_tsql(state, msg="Incorrect result."):
+    """High level function which wraps other SCTs for checking results."""
+
+    stu_res = state.student_result
+    sol_res = state.solution_result
+
+    # empty test
+    test_has_columns(state)
+    # row test
+    test_nrows(state)
+    # column tests
+    child = _sort_columns_indiv(state)
+    for k in sol_res:
+        msg = "Column `{}` in the solution does not have a column with matching values in your results."
+        test_column(child, k, msg, match = 'exact')
```

Comment:
It looks like match was changed from "any" to "exact". This is different from what's being used in the tsql course, and was changed to "any" in response to Graeme's feedback. Are we changing it back? 

Original code: https://github.com/datacamp/sqlwhat/blob/msft-check-result/sqlwhat/checks/check_result.py#L34